### ISSUE TITLE
Session Driver files

### ISSUE THREAD

**Issue by Mindhk92** [bcit-ci/CodeIgniter#4770]
Using Session ` $config['sess_driver'] = 'files';`
With client from Chrome on Android, 
the session always empty when redirected to another page.

**Comment by narfbg** [comment]
Post this on the [forums](http://forum.codeigniter.com/) please - it barely says anything, we can't take it seriously as a bug report.

**Comment by Mindhk92** [comment]
My Apologies, narfbg.

I've found the problem, but still don't know why.

When I enable profiler, in the  HTTP HEADERS, the REMOTE_ADDR somehow showing Google IP Address instead of my current Public IP Address, but HTTP_X_FORWARDED_FOR showing my current Public IP Address. 

It appear, when the data saver is on, the session could not be load.
When I set it off, the session appear when I var_dump it and the REMOTE_ADDR showing my current IP .
